lib/screens/enlistment/enlistment_page.dart

```diff
@@ -12,18 +14,140 @@ class _EnlistmentPageState extends State<EnlistmentPage> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(),
-      body: Column(
-        children: [
-          Text('this is the enlistment page'),
-          ElevatedButton(
-            onPressed: () {
-              Navigator.of(context).push(
-                MaterialPageRoute(builder: (_) => EnlistmentClassesPage()),
-              );
-            },
-            child: Text('view semester'),
-          ),
-        ],
+      body: Container(
+        padding: EdgeInsets.fromLTRB(16, 48, 16, 29),
+        child: Column(
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                Text('Enlistment Preparer',
+                    style: Theme.of(context)
+                        .textTheme
+                        .headline4
+                        .copyWith(color: AppColors.GRAY_DARK[0])),
+                CircleAvatar(
```

Comment:
you can switch to `HelpButton` or whatever the class name is, from `lib/widgets/help_...`